cfme/tests/automate/test_instance.py

```diff
@@ -219,3 +220,63 @@ def test_automate_relationship_trailing_spaces(request, klass, namespace, domain
             assert result.output == ""
         else:
             assert search in result.output
+
+
+@pytest.fixture(scope="module")
+def copy_instance(domain):
+    """
+    This fixture copies the instance '/ManageIQ/System/Request/ansible_tower_job' to new domain.
+    """
+    # Instantiating class 'Request'
+    klass = (
+        domain.parent.instantiate(name="ManageIQ")
+        .namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+    )
+
+    # Instantiating instance 'ansible_tower_job' and copying it to new domain
+    klass.instances.instantiate(name="ansible_tower_job").copy_to(domain.name)
+    instance = (
+        domain.namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+        .instances.instantiate(name="ansible_tower_job")
+    )
+    yield instance
```

Comment:
Optional:
please check domain deletion proper or not at tear-down? Otherwise this stuff will not delete at tear-down.

Reply:
Domain is deleted successfully which leads to deletion of stuff created under it.